cmd/jira-lifecycle-plugin/server_test.go

```diff
@@ -356,7 +356,7 @@ func TestHandle(t *testing.T) {
 		options                    JiraBranchOptions
 		fullConfig                 Config
 		expectedLabels             []string
-		expectedComment            string
+		expectedComments           []string
```

Comment:
This change in type is unnecessary as `handle` only ever creates 1 comment. IMO it would be better to have keep this as `string` and change the `checkComments` line at the end of the function to use `[]string{tc.expectedComment}`.